KEYB.ASM

```diff
@@ -0,0 +1,159 @@
+;-----------------------------------------------------------
+; 32-bit Keyboard functionality.
+;-----------------------------------------------------------
+IDEAL
+P386
+MODEL FLAT, C
+ASSUME cs:_TEXT,ds:FLAT,es:FLAT,fs:FLAT,gs:FLAT
+
+INCLUDE "keyb.inc"
+
+KEY_BUFFER	EQU 60h			; the port of the keyboard buffer
+KEY_CONTROL	EQU 61h			; the port of the keyboard controller
+PIC_PORT	EQU 20h			; the port of the peripheral
+
+;-----------------------------------------------------------
+; CODE
+;-----------------------------------------------------------
+CODESEG
+
+; Installs the custom keyboard handler
+PROC __keyb_install_keyboard_handler
```

Comment:
Please rename `PROC` and `ENDP` to `proc` and `endp`. This is syntactically the same in TASM but it breaks the conversion script. I could update the conversion script but our entire codebase uses the lowercase versions so it seems better to keep this consistent.